src/main/java/com/google/javascript/gents/TypeAnnotationPass.java

```diff
@@ -108,6 +108,25 @@ public void visit(NodeTraversal t, Node n, Node parent) {
             setTypeExpression(n, bestJSDocInfo.getReturnType(), true);
           }
           break;
+        case CLASS:
+          if (bestJSDocInfo != null) {
+            List<JSTypeExpression> interfaces = bestJSDocInfo.getImplementedInterfaces();
+            if (!interfaces.isEmpty()) {
+              Node impls = new Node(Token.IMPLEMENTS);
+              for (JSTypeExpression type : interfaces) {
+                impls.addChildToBack(convertTypeNodeAST(type.getRoot()));
```

Comment:
@Implements sorry, I accidentally mentioned you, I was talking about a bit of code that literally contains the string `@implements`. Nevermind :-)

@gmoothart added a comment. It's converting the `@implements {...}` JSDoc comments to TypeScript `class A implements B, C {}`.